### DISCUSSION TITLE
What is my success/cancelUrl?

### DISCUSSION THREAD

**Discussion by BjoernRave** [A-Tokyo/react-native-stripe-checkout-webview#23] [Q&A] [answered]
Do I need to use a deep link as the success url to get back to my app?

**Comment by a-tokyo** [comment, marked as answer]
your success url  can be:
https://your_website.com/stripe_checkout_success?sc_checkout=success&sc_sid={CHECKOUT_SESSION_ID}

No, you don't need to use a deeplink since we don't leave the app from the first place.
